src/movies.js

```diff
@@ -1,16 +1,132 @@
 // Iteration 1: All directors? - Get the array of all directors.
+function getAllDirectors(movies){
+       //code
+       let answer = movies.map((movie)=>{
+           return movie.director;
+       })
+       return answer;
+}
 // _Bonus_: It seems some of the directors had directed multiple movies so they will pop up multiple times in the array of directors. How could you "clean" a bit this array and make it unified (without duplicates)?
 
 // Iteration 2: Steven Spielberg. The best? - How many drama movies did STEVEN SPIELBERG direct?
-
+function howManyMovies(movies){
+    let dramaAndSteven;
+    let answer = movies.filter((movie)=>{
+        for(let i=0; i<movie.genre.length; i++){
```

Comment:
The loop works good for this, but there is a method called .includes that looks for an element inside an array, so by using that you could avoid the loop. Will look more clean.